app/views/admin/admin_users/show.html.erb

```diff
@@ -4,7 +4,9 @@
   <!--/descriptions-->
 
   <!--subnav-->
+  <p class="note">To fix common errors with this user's Subscriptions and Stats pages, and to reindex their works and bookmarks, choose "Troubleshoot Account."<p>
```

Comment:
Fine with me as long as it won't make it harder to track down the strings! (I assumed we were going to search for `ts('')` when it was time for us to make our switch.)